src/core/selfcare_be.tf

```
@@ -0,0 +1,153 @@
+module "selfcare_be" {
+  source = "git::https://github.com/pagopa/azurerm.git//app_service?ref=v2.0.2"
+
+  name = "${var.location}-${var.env_short}"
```

```suggestion
  name = format("%s-app-devportal-be", local.project)
```
missing app service name
we don't use location but we can introduce it. Better to add a `var.location_short`